### DISCUSSION TITLE
Samsung AIR

### DISCUSSION THREAD

**Discussion by markkelynack** [airsdk/Adobe-Runtime-Support#2443] [General]
Just a general question out of curiosity, Does Samsung use AIR in any of their products?

**Comment by ajwfrost** [comment]
That would be a question to ask Samsung ... they may own Harman, but there is still an operational separation between the companies. Lots of collaborative areas of course, but they're not driving our Adobe-related business, this all originated from the former "Teleca" company (scaling partner of Adobe since 2006, we were bought by Symphony Technology Group to form "Symphony Teleca Corporation", which was then bought by Harman).

What I would say is that our Adobe-related business prior to the 2019 agreement was to put Flash/AIR technologies into various embedded platforms and devices, including automotive (IVI) and TV/STB platforms (and also supermarket scales, ovens, printers, etc...)

thanks
